setup/docker/build.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+
+which mvn &> /dev/null || { echo >&2 "Maven package cant be found on path. Aborting."; exit 1; }
+which docker &> /dev/null  || { echo >&2 "Docker package cant be found on path. Aborting."; exit 1; }
+mvn package || { echo >&2 "Maven package has failed. Aborting."; exit 1; }
+
+export company="tananaev"
+export software="traccar"
+export version=$(head -n 10 ./pom.xml |grep version|cut -d ">" -f2|cut -d"<" -f1)
+
+tmp="./setup/docker/tmp"
+
+mkdir -p ${tmp}/data/database
```

Comment:
I mean we need to update https://www.traccar.org/docker/ and its not on this project.
Replace this:
docker run -d --name traccar-server -v /my-database-folder:/opt/traccar/data/database -p 5000-5150:5000-5150 -p 8082:8082 tananaev/traccar:[VERSION]

With this:
docker run -d --name traccar-server -v /my-database-folder:/opt/traccar/data -p 5000-5150:5000-5150 -p 8082:8082 tananaev/traccar:[VERSION]